### ISSUE TITLE
Test runner update for GraphQL

### ISSUE THREAD

**Issue by gvolfing** [memgraph/memgraph#1007]
The test runner machines should be able to set up the Neo4j GraphQL library.

A couple of `npm` packages would be needed, The tool-chain might also has to be modified for this to work.

https://neo4j.com/developer/graphql/#_using_the_neo4j_graphql_library_with_neo4j_auradb

**Comment by matea16** [comment]
[+1 from community](https://github.com/memgraph/memgraph/issues/2506) for the support of the "apoc.date.convertFormat" procedure

**Comment by jroith** [comment]
> [+1 from community](https://github.com/memgraph/memgraph/issues/2506) for the support of the "apoc.date.convertFormat" procedure

The GraphQL library appear to always call convertFormat with two constant parameters, so a partial implementation that throws for other parameters would initially be sufficient:

`apoc.date.convertFormat(str, "iso_zoned_date_time", "iso_offset_date_time")`

**Comment by katarinasupe** [comment]
@gitbuda just a note that there has been some activity on this issue, along with https://github.com/memgraph/mage/issues/525 reported

**Comment by gitbuda** [comment]
Yes, recently it seems like we have more reported issues under the GraphQL compatibility -> I'll try to organize a project to improve that in the short run ⏳

**Comment by gitbuda** [comment]
WIP under https://github.com/memgraph/memgraph/pull/3027
